Add documentation for Custom timeframe on Monitors (#9140)
DataDog/documentation: content/en/monitors/monitor_types/metric.md

* The `average`, `maximum`, `minimum`, or `in total`
* of the `change` or `% change`

> * over `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 48h

Reviewer: ```suggestion
* over `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 48 hours.
```

> * over `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 48 hours.

Reviewer 2: ```suggestion
* over `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 48 hours.
```
missed one

> * compared to `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 48h before

Reviewer: ```suggestion
* compared to `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 48 hours before.
```
